### PR TITLE
add UpdateShardCount support for Kinesis

### PR DESCRIPTION
**Pull request by benvandenberg**
Shamelessly pilfered from https://github.com/aventurella/localstack/commit/0a1b454, this commit addresses https://github.com/localstack/localstack/issues/654

This commit allows Terraform to run against Localstack and create Kinesis resources.

### CODE REVIEW

File: localstack/services/kinesis/kinesis_listener.py [hunk 1]
```diff
@@ -59,6 +60,29 @@ def return_response(self, method, path, data, headers, response):
...
+            #
+            response = Response()
+            response.status_code = 200
+            content = {
+                'CurrentShardCount': 1,
+                'StreamName': data['StreamName'],
+                'TargetShardCount': data['TargetShardCount']
+            }
+            response.encoding = 'UTF-8'
+            response.headers = headers
```
Comment (whummer) [inline localstack/services/kinesis/kinesis_listener.py]: Not sure this is needed/appropriate here - `headers` contains the _request_ headers, not _response_ headers. I think line 82 should be removed.

**Comment by benvandenberg** [comment]
I will make the change based on your comment and use the change from @aventurella as the basis for the branch

**Comment by benvandenberg** [comment]
Any possibility that this can get merged?

**Comment by whummer** [comment]
Thanks again! 👍

**Pull request merged**